### PR TITLE
add pt-br/docs/reference/issues-security/_index.md

### PR DESCRIPTION
**Pull request by italofernandes13**
Translate docs to pt-br

docs/reference/issue-security/_index.md

Related issues
#13939

### CODE REVIEW

File: content/pt-br/docs/reference/issues-security/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Problemas e Segurança do Kubernetes
+weight: 40
+toc-hide: true
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/issues-security/_index.md:4]: This does not exist in the original version. It should not be added in the translation.

**Review by stormqueen1990 — COMMENTED** [review]
Hi there, @italofernandes13!

Thank you very much for your contribution! 🎉 

Right now, this translation does not seem on par with the English version. Could you please review it and update accordingly, so the remainder of the content (including weight) matches the original version?

Thanks in advance!

/hold

**Comment by stormqueen1990** [comment]
Please feel free to reach out on the `#kubernetes-docs-pt` channel of Kubernetes Slack should you have any questions!

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**